packages/api-server/api_server/routes/tasks/tasks.py

```diff
@@ -186,6 +186,7 @@ async def post_activity_discovery(
 async def post_cancel_task(
     request: mdl.CancelTaskRequest = Body(...),
 ):
+    logger.info(request)
```

Comment:
This log can be very messy/noisy as we can't identify the source, although in this case I don't think we are using `CancelTaskRequest` any where else so it is technically traceble. For now should be ok but we should have a better logging framework in the future.

Reply:
Yup, for sure, we will migrate to use the logging server or anything equivalent later on. This is just a project requirement for now, that we should be able to tell who cancelled tasks.